core/src/main/java/com/datastax/oss/driver/api/core/type/reflect/GenericType.java

```diff
@@ -226,6 +226,24 @@ public final boolean isPrimitive() {
     return where(freeVariable, GenericType.of(actualType));
   }
 
+  /**
+   * Returns the raw type of {@code T}. Formally speaking, if {@code T} is returned by {@link
+   * java.lang.reflect.Method#getGenericReturnType}, the raw type is what's returned by {@link
+   * java.lang.reflect.Method#getReturnType} of the same method object. Specifically:
+   *
+   * <ul>
+   *   <li>If {@code T} is a {@code Class} itself, {@code T} itself is returned.
+   *   <li>If {@code T} is a parameterized type, the raw type of the parameterized type is returned.
+   *   <li>If {@code T} is an array type , the returned type is the corresponding array class. For
+   *       example: {@code List<Integer>[] => List[]}.
+   *   <li>If {@code T} is a type variable or a wildcard type, the raw type of the first upper bound
+   *       is returned. For example: {@code <X extends Foo> => Foo}.
+   * </ul>
+   */
+  public Class<?> getRawType() {
```

Comment:
Shouldn't this be `<? super T>`?

Reply:
you are right, good catch!  I'll fix it.